### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by jdnichollsc** [fahrmairm/ionic-gulp-image-task#1]
I'm trying your module but it doesn't work :/

See please: http://screencast.com/t/1QP36Avv

PD: I'm using =>

```
gulp.task('images', function(){
  return imageCopyOptimize({ 
    src: './app/img/**/*',
    dest: 'www/img'
  });
});
```

Regards, Nicholls

**Comment by fahrmairm** [comment]
upps, github was not up to date with the latest version. Can you please check if you use 1.0.1? Other than that the screenshot looks like the gulp-image dependency (or one of its dependencies) is not correctly installed or working. I have not tested it under windows so far. I can do that tomorrow when I have access to a windows vm.

Michael

**Comment by achembarpu** [comment]
Yep, I'm facing the same issue. However, like @fahrmairm says, it seems to be a `gulp-image` issue, as removing that part from the pipeline makes sure the task runs fine. I'm debugging it to find out what's going wrong.

**Comment by achembarpu** [comment]
`imagemin` built and installed successfully on my system (windows 10), but it seems to be exiting without throwing any errors. Not sure how to proceed.
